ports/espressif/esp-idf-config/sdkconfig-debug.defaults

```diff
@@ -15,6 +15,8 @@ CONFIG_COMPILER_OPTIMIZATION_SIZE=y
 #
 CONFIG_ESP_SYSTEM_PANIC_PRINT_HALT=y
 CONFIG_ESP_PANIC_HANDLER_IRAM=y
+CONFIG_ESP_CONSOLE_SECONDARY_NONE=y
+# CONFIG_ESP_CONSOLE_SECONDARY_USB_SERIAL_JTAG is not set
```

Comment:
Is this new in 5.3.1, or does it fix some existing issue?

Reply:
I think 5.3 switched the default for USB_SERIAL_JTAG to true. I've been setting this manually for debug builds before.